build-test-local.xml

```diff
@@ -306,6 +306,7 @@ local.bundles.backup.file=${liferay.home.parent.dir}/bundles-master-${git.commit
 	</target>
 
 	<target name="setup-custom-suite">
+	<property name="integration" value="false" />
```

Comment:
SF failure is here. Needs to indented. Thanks!

Reply:
FIxed!